PublicationsRouter/v2/api/API.md

```diff
@@ -222,6 +274,21 @@ Note the last of these is different from the Validation endpoint.
         "location" : "<url path for api endpoint for newly created notification>"
     }
 ```
+
+-- Notification List
+```
+    HTTP 1.1  202 Accepted
+    Content-Type: application/json
+
+    {
+        "successful": 3
+        "success_ids": [1, 3, 5],
```

Comment:
I think you should use placeholders rather than example values, which would explain what the values mean.